.github/deploy/Utilities/Databricks/New-DatabricksScope.ps1

```diff
@@ -0,0 +1,35 @@
+function New-DatabricksScope {
+    param (
+      [Parameter(Mandatory = $true)]
+      [ValidateNotNullOrEmpty()]
+      [string]
+      $name,
+
+      [Parameter(Mandatory = $false)]
+      [ValidateNotNullOrEmpty()]
+      [string]
+      $keyVaultDns,
+
+      [Parameter(Mandatory = $false)]
+      [ValidateNotNullOrEmpty()]
+      [string]
+      $keyVaultResourceId
+    )
+
+    $json = Convert-Safe-FromJson -text (databricks secrets list-scopes --output JSON)
+    if($json.scopes){
+      $scopes = $json | Select-Object -expand "scopes" | Select-Object -expand "name"
+      if ($scopes -contains $name) {
+        Write-Host "  The scope '$name' already exists" -ForegroundColor DarkYellow
+      }
+    }
```

Comment:
Is this correct?

If json.scopes has some kind of scope "scopeA". Then the if statement is true. 
The scopes is then extracted to $scopes. BUT, there is no "else"-statement to the "if ($scopes -contains $name)".

What could then happen?
If there should be created "scopeA", and there exists a "scopeB". Then, the algorithm will find that there is scopes, but it does not contain the "scopeA". It should then be created, but it doesnt!